invrec_deutanal.py

```diff
@@ -0,0 +1,290 @@
+    d_ph0 = d['indirect',4].C.sum('t2')
+    d_ph0 /= abs(d_ph0)
+    d['indirect',4] /= d_ph0
+    
+    ph1_5dim = 5.729
+    d['indirect',5] *= exp(-1j*2*pi*ph1_5dim/SW*d['indirect',5].fromaxis('t2'))
+    d_ph0 = d['indirect',5].C.sum('t2')
+    d_ph0 /= abs(d_ph0)
+    d['indirect',5] /= d_ph0
+    
+    ph1_6dim = -0.091
+    d['indirect',6] *= exp(-1j*2*pi*ph1_6dim/SW*d['indirect',6].fromaxis('t2'))
+    d_ph0 = d['indirect',6].C.sum('t2')
+    d_ph0 /= abs(d_ph0)
+    d['indirect',6] /= d_ph0
+
+    ph1_7dim = 1.959
+    d['indirect',7] *= exp(-1j*2*pi*ph1_7dim/SW*d['indirect',7].fromaxis('t2'))
+    d_ph0 = d['indirect',7].C.sum('t2')
+    d_ph0 /= abs(d_ph0)
+    d['indirect',7] /= d_ph0
+    #quit()
+
+    ph1_8dim = -0.846
+    d['indirect',8] *= exp(-1j*2*pi*ph1_8dim/SW*d['indirect',8].fromaxis('t2'))
+    d_ph0 = d['indirect',8].C.sum('t2')
+    d_ph0 /= abs(d_ph0)
+    d['indirect',8] /= d_ph0
+    #quit()
+
+    ph1_9dim = 4.670
+    d['indirect',9] *= exp(-1j*2*pi*ph1_9dim/SW*d['indirect',9].fromaxis('t2'))
+    d_ph0 = d['indirect',9].C.sum('t2')
+    d_ph0 /= abs(d_ph0)
+    d['indirect',9] /= d_ph0
+    #quit()
+
+    ph1_10dim = -0.976
+    d['indirect',10] *= exp(-1j*2*pi*ph1_10dim/SW*d['indirect',10].fromaxis('t2'))
+    d_ph0 = d['indirect',10].C.sum('t2')
+    d_ph0 /= abs(d_ph0)
+    d['indirect',10] /= d_ph0
+    #quit()
+
+    ph1_11dim = -0.812
+    d['indirect',11] *= exp(-1j*2*pi*ph1_11dim/SW*d['indirect',11].fromaxis('t2'))
+    d_ph0 = d['indirect',11].C.sum('t2')
+    d_ph0 /= abs(d_ph0)
+    d['indirect',11] /= d_ph0
+    #quit()
+
+    ph1_12dim = -0.941
+    d['indirect',12] *= exp(-1j*2*pi*ph1_12dim/SW*d['indirect',12].fromaxis('t2'))
+    d_ph0 = d['indirect',12].C.sum('t2')
+    d_ph0 /= abs(d_ph0)
+    d['indirect',12] /= d_ph0
+    #quit()
+
+    ph1_13dim = 5.695
+    d['indirect',13] *= exp(-1j*2*pi*ph1_13dim/SW*d['indirect',13].fromaxis('t2'))
+    d_ph0 = d['indirect',13].C.sum('t2')
+    d_ph0 /= abs(d_ph0)
+    d['indirect',13] /= d_ph0
+    #quit()
+
+    ph1_14dim = -1.133
+    d['indirect',14] *= exp(-1j*2*pi*ph1_14dim/SW*d['indirect',14].fromaxis('t2'))
+    d_ph0 = d['indirect',14].C.sum('t2')
+    d_ph0 /= abs(d_ph0)
+    d['indirect',14] /= d_ph0
+    #quit()
+
+    ph1_15dim = -1.046
+    d['indirect',15] *= exp(-1j*2*pi*ph1_15dim/SW*d['indirect',15].fromaxis('t2'))
+    d_ph0 = d['indirect',15].C.sum('t2')
+    d_ph0 /= abs(d_ph0)
+    d['indirect',15] /= d_ph0
+    #quit()
+    print(ndshape(d))
+    #d['indirect',0] *= -1
+    #d['indirect',1] *= -1
+    #d['indirect',2] *= -1
+    #d['indirect',3] *= -1
+    #d['indirect',4] *= -1
+    #d['indirect',5] *= -1
+    #d['indirect',6] *= -1
+    d['indirect',7] *= -1
+    d['indirect',8] *= -1
+    d['indirect',9] *= -1
+    d['indirect',10] *= -1
+    d['indirect',11] *= -1
+    d['indirect',12] *= -1
+    d['indirect',13] *= -1
+    d['indirect',14] *= -1
+    d['indirect',15] *= -1
```

Comment:
this cannot be part of the pull request.  you need to get rid of this and replace it with something automatic.